src/components/skills-quiz/tests/SearchCourseCard.test.jsx

```diff
@@ -132,25 +128,30 @@ const SearchCourseCardWithContext = ({
 
 describe('<SearchCourseCard />', () => {
   test('renders the correct data', async () => {
-    let containerDOM = {};
-    await act(async () => {
-      const { container } = renderWithRouter(
-        <SearchCourseCardWithContext
-          index={testIndex}
-        />,
-      );
-      containerDOM = container;
-    });
+    const { container, history } = renderWithRouter(
```

Comment:
[learned] Didn't know you could also extract out history from the renderWithRouter function. Will keep this in mind